src/note.h

```diff
@@ -26,10 +26,13 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 */
 
-#pragma once
+#if !defined(H_NOTE)
```

Comment:
I'd prefer `#ifndef`, it lines up with `#define` and is more idiomatic.

```suggestion
#ifndef H_NOTE
```